docs/en/observability/exploratory-data-visualizations.asciidoc

```diff
@@ -1,28 +1,201 @@
+. To add the visualization to an existing case, click *Add to case* and select the correct case.
+
+[discrete]
+[[metrics]]
+== Metrics
+
+Based on your infrastructure metrics, you can create various visualizations relating to
+memory or CPU usage, or any of the <<data-types>>[available data types].
+
+[role="screenshot"]
+image::images/exploratory-view-metrics.png[Explore data for Infra metrics]
+
+|===
+
+| *KPI over time* | The KPI over time histogram represents the performance indicators based on
+the metric you select.
+
+By default, the `system CPU usage` metric is selected.
+
+|===
+
+[discrete]
+[[explore-data-synthetics]]
+== Synthetic Monitoring
+
+Based on the Uptime data you are sending to your deployment, you can create various visualizations relating to
+monitor durations, pings over time, or any of the <<data-types>>[available data types].
+
+[role="screenshot"]
+image::images/exploratory-view-uptime.png[Explore data for Uptime]
+
+|===
+
+| *Monitor duration* | The Uptime monitor duration time-series chart displays the timing for each check that {heartbeat} performed.
+
+This visualization helps you to gain insights into how quickly requests resolve by the targeted endpoint
+and give you a sense of how frequently a host or endpoint was down in your selected timespan.
+
+| *Pings histogram* | The Uptime pings chart is a graphical representation of the check statuses over time.
+
+|===
+
+[discrete]
+[[explore-data-user-experience]]
+== User Experience
+
+Based on the {user-experience} data from your instrumented applications, you can create
+detailed visualizations for performance distributions, key performance indicators (KPI) over time,
+and for core web vitals of your web applications.
+
+[role="screenshot"]
+image::images/exploratory-view-ux-page-load-time.png[Explore data for User Experience - page load time]
+
+|===
+
+| *KPI over time* | The KPI over time histogram represents the performance indicators based on
+the metric you select.
+
+By default, the `page views` metric is selected.
+
+| *Performance distribution* | The Performance distribution time-series chart enables you to examine the perceived
+performance of your web applications based on the metric you select.
+
+By default, the `page load time` metric is selected.
+
+| *Core web vitals* | The Core web vitals chart is a graphical representation of key metrics, such as
+loading performance, load responsiveness, and visual stability, for each of your web applications.
+
+By default, the `largest contentful paint` metric is selected. Hover over the chart to display crosshairs
+with performance indicators for each web application: `poor`, `average`, and `good`.
+
+|===
+
+[discrete]
+[[data-types]]
+== What data types can I explore?
+
+The following table shows which data types are available for each report type:
+
+|===
+|Data type | Synthetics monitoring | Infra metrics | User experience (RUM) | Mobile experience
+
+| Monitor duration | {y} | | |
```

Comment:
These attributes aren't resolving. You probably need to turn on asciidoc resolution (set `a` in `cols`)

Reply:
This one is a little strange...I copied a table from Fleet that does resolve the icons (no `a` refs as far as I can see) and this builds ok locally within Fleet but exact same table within this Obs page doesn't. 

I did the reverse and put this Obs table into Fleet guide and it renders ok so not sure where the problem is...

Also tried adding the `a` to columns as per https://docs.asciidoctor.org/asciidoc/latest/tables/format-column-content/ but still not rendering...

Reply:
Ah cool, thanks - not a rabbit hole, a learning break ;) good to dig into some of these and learn more about the builds.

So basically we hadn't used that kind of table in main Obs Guide yet?

Reply:
![image](https://user-images.githubusercontent.com/89071729/158641292-a22bdcce-3b62-4489-98ec-3592e88e0569.png)